server/enterprise/message_export/shared/export_data.go

```diff
@@ -39,24 +37,24 @@ type PostExport struct {
 
 	// when a message is edited, the EditedOriginalMsg points to the message Id that now has the newly edited message.
 	EditedNewMsgId string
-	Message        string // the text body of the post
-	PreviewsPost   string // the post id of the post that is previewed by the permalink preview feature
+	Message        string          // the text body of the post
```

Comment:
nit: Do you know why this was not attached to `model.PostExport` in the first place? It could simply be empty for other types.

Reply:
@isacikgoz  Sorry, what specifically are you referring to, the Message field? It always was attached to PostExport.

Reply:
Yeah, wondering why the `Message` field is not part of `model.MessageExport`, seems like it should be part of it. Anyways, it's a very minor thing.

Reply:
@isacikgoz  Ah, gotcha. It's because we modify this `Message` field, and we don't want to modify the original `model.MessageExport` message.